### PR TITLE
UI/about

### PR DESCRIPTION
**Pull request by america-nishi**
info added

### CODE REVIEW

File: src/features/channels/components/channel-list/about/about.tsx [hunk 1]
```diff
@@ -1,20 +1,42 @@
...
+};
+
 const About = ({ title, message, onClick }: Props) => {
-  const [open, setOpen] = React.useState(false);
+  const [user, setUser] = useState<WebAppInfo[]>();
+  const [open, setOpen] = useState(false);
   const styles = useStyles();
+  const key = 0;
+
+  useOnMount(() => {
```
Comment (AlefrankM) [inline src/features/channels/components/channel-list/about/about.tsx:35]: Why not write all in a single line? I think so is possible and equally readable

**Review by AlefrankM — APPROVED** [review]
Good job

File: src/features/channels/components/channel-list/about/about.tsx [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
 };
 
+const loadWebAppInfo = async (): Promise<WebAppInfo[]> => {
+  const response = await httpService.get<WebAppInfo[]>('/v1/api/version');
+  return response.data;
+};
+
 const About = ({ title, message, onClick }: Props) => {
-  const [open, setOpen] = React.useState(false);
+  const [user, setUser] = useState<WebAppInfo[]>();
```
Comment (josetheis-stackit) [inline src/features/channels/components/channel-list/about/about.tsx:30]: Why is this state called user? We're setting this to the application's info, not users. Try to use a better name for the state so the code is easier to understand

File: src/features/channels/components/channel-list/about/about.tsx [hunk 3]
```diff
@@ -42,7 +64,33 @@ const About = ({ title, message, onClick }: Props) => {
...
         <DialogContent>
-          <DialogContentText id="alert-dialog-description">{message}</DialogContentText>
+          <DialogContentText id="alert-dialog-description">
+            <Grid className={styles.infoBox}>
+              <Divider />
+
+              <Grid className={styles.infoLine}>
+                <Grid className={styles.infoLineLeft}>{user && user[0].name}</Grid>
+                <Grid className={styles.infoLineRight}>{user && user[0].version}</Grid>
+              </Grid>
```
Comment (josetheis-stackit) [inline src/features/channels/components/channel-list/about/about.tsx:74]: Use the map function to iterate the array. example: user.map((item) => <Grid className={styles.infoLineLeft}>{item.name}</Grid>)

File: src/features/channels/components/channel-list/about/about.tsx [hunk 4]
```diff
@@ -1,20 +1,42 @@
...
+
 type Props = {
   title: string;
   message: string;
   onClick?: () => void;
 };
 
+const loadWebAppInfo = async (): Promise<WebAppInfo[]> => {
+  const response = await httpService.get<WebAppInfo[]>('/v1/api/version');
+  return response.data;
```
Comment (josetheis-stackit) [inline src/features/channels/components/channel-list/about/about.tsx:26]: Just a suggestion, you can destructure the response object and return the data directly. Ex: 
```suggestion
const { data } = await httpService.get<WebAppInfo[]>('/v1/api/version');
return data;
```

**Review by josetheis-stackit — APPROVED** [review]
Left some comments. Take a look when you can.

File: src/features/channels/components/channel-list/about/about.tsx (hunk 1 above)
Comment (america-nishi) [reply src/features/channels/components/channel-list/about/about.tsx:35]: Ok, I will fix it. I see I'm not using key so I will delete it.

File: src/features/channels/components/channel-list/about/about.tsx (hunk 2 above)
Comment (america-nishi) [reply src/features/channels/components/channel-list/about/about.tsx:30]: That's correct I will fix it.

File: src/features/channels/components/channel-list/about/about.tsx (hunk 3 above)
Comment (america-nishi) [reply src/features/channels/components/channel-list/about/about.tsx:74]: Ok, I will

File: src/features/channels/components/channel-list/about/about.tsx (hunk 4 above)
Comment (america-nishi) [reply src/features/channels/components/channel-list/about/about.tsx:26]: I will consider it

File: src/features/channels/components/channel-list/about/about.tsx [hunk 5]
```diff
@@ -42,7 +63,24 @@ const About = ({ title, message, onClick }: Props) => {
       >
         <DialogTitle id="alert-dialog-title">{title}</DialogTitle>
         <DialogContent>
-          <DialogContentText id="alert-dialog-description">{message}</DialogContentText>
+          <DialogContentText id="alert-dialog-description">
+            <Grid className={styles.infoBox}>
+              <Divider />
+              {info &&
+                info.map((item) => {
```
Comment (AlefrankM) [inline src/features/channels/components/channel-list/about/about.tsx:70]: You can destructure the object like this and return directly the component
```suggestion
                info.map(({ name, version }) => 
                    <Grid>
                    ...
                     </Grid>
                    
```

**Review by AlefrankM — APPROVED** [review]
Reviewed

**Pull request merged**